field/ShipBFieldMap.cxx

```diff
@@ -127,8 +127,9 @@ void ShipBFieldMap::Field(const Double_t* position, Double_t* B)
     // 4. x < 0, y < 0: Bx = Bx
 
     Float_t BxSign(1.0);
+    Float_t BySign(1.0);
```

Comment:
In the existing code, the use case for isSymmetric_ is described as:
 Bx is antisymmetric in x and y, By is symmetric and Bz has no symmetry

if you have a different use case, you need to open another if statement and add another configuration variable. 

To avoid these complications, I would rather suggest that you resolve this symmetry when you create the field map file, explicitly give the field values for all combinations of x and y, negative and positive.

Reply:
I'll replace the field file by the new one with By inverted.

Reply:
Ok, new fieldmap file is ready. What is our plan for our moves in others directions?

Reply:
> Ok, new fieldmap file is ready. What is our plan for our moves in others directions?

Super, please make a new commit where this file is restored back and provide link for new file. The changes should be visible in this discussion.